tss-esapi/src/context/tpm_commands/context_management.rs

```diff
@@ -163,23 +163,23 @@ impl Context {
     ///            specified in [Provision].
     /// * `object_handle` - The handle of a loaded object.
     /// * `persistant` - If the `object_handle` is transient object then this
```

Comment:
Is the parameter in the code spelled incorrectly as well?

Reply:
:thinking: yeah it seems so... it seems codespell ignores things between ` and ` (considers them as raw names perhaps)... I'll fix that one manually.

Reply:
So it only cares about comments without backticks not things that are in the actual code?

Reply:
It checks for common misspellings but I guess there is a heuristic that disables the use for the actual code or stuff that looks like code (I've removed backticks and it corrected that). The advantage is that the false-positives are rare, but the downside - as we saw - is that it doesn't catch everything :-/

(FTR more details are at: https://github.com/codespell-project/codespell#codespell)